src/ansys/fluent/core/services/field_data.py

```diff
@@ -415,35 +430,53 @@ def __init__(
             self.add_pathlines_fields_request,
         )
 
+    @staticmethod
+    def _data_type_convertor(old_args_dict, new_args):
+        d_type_list = []
+        if old_args_dict.get("provide_vertices"):
+            d_type_list.append(SurfaceDataType.Vertices)
+        if old_args_dict.get("provide_faces"):
+            d_type_list.append(SurfaceDataType.FacesConnectivity)
+        if old_args_dict.get("provide_faces_centroid"):
+            d_type_list.append(SurfaceDataType.FacesCentroid)
+        if old_args_dict.get("provide_faces_normal"):
+            d_type_list.append(SurfaceDataType.FacesNormal)
+        return {new_args[0]: d_type_list}
+
+    @deprecate_arguments(
+        old_args=["surface_names", "surface_ids"],
+        new_args=["surfaces"],
```

Comment:
If the converters are acting on the entire dictionaries then specifying these lists is redundant, which is evident in the converter implementations.